docs/zkapps/o1js-reference/classes/Unconstrained.md

```diff
@@ -0,0 +1,195 @@
+[o1js](../README.md) / [Exports](../modules.md) / Unconstrained
+
+# Class: Unconstrained<T\>
+
+Container which holds an unconstrained value. This can be used to pass values
```

Comment:
```suggestion
Container which holds an unconstrained value. You can use this to pass values
```

Reply:
I don't know if we want to make the API docs actionable, but if we did, this would be a valid suggestion